packages/bpk-component-modal/src/bpk-modal-dialog.scss

```diff
@@ -98,9 +98,11 @@
 
   &__dialog-heading {
     margin: 0;
-    font-weight: 700;
 
     @include bpk-text-base;
```

Comment:
To be clear this _is_ a breaking change for `@include bpk-text-base;` right? we'll have to provide upgrade instructions along the lines of:
```
// before
.my-style {
  font-weight: bold;
  
  @include bpk-text-base;
}

// after
.my-style {
  @include bpk-text-base;
  @include bpk-text-base--bold;
}
```

Right?